cypher-editor-support/src/CypherEditorSupport.js

```diff
@@ -42,16 +43,55 @@ export class CypherEditorSupport {
   referencesProviders = {};
   completion = new AutoCompletion();
   queriesAndCommands = [];
+  listeners = [];
+  version = null;
 
   constructor(input = '') {
     this.update(input);
   }
 
-  update(input = '') {
+  getVersion() {
```

Comment:
You don't need this method. It's not used from outside, because `ensureVerrsion` is de way.